js/datatables.js

```diff
@@ -301,7 +363,14 @@ function sdkcall(svc, method, params, alert_on_errors, backoff) {
                     reject(data);
                     return;
                 }
-                
+                console.log(`CloudFormationer log - sdkcall with svc: ${svc}, method: ${method}, params: ${params}`)
+                console.log('CloudFormationer log - sdkcall data before filtering: ', data)
```

Comment:
please log also the count of resources before the filter and after the filter

Reply:
It's pretty obvious the count from the current logs - 
And adding the count of the logs can be problematic as in every service the list is located in another place.
![image](https://user-images.githubusercontent.com/57212002/177332306-c5ed272a-cb29-4bd7-93a6-ac17be074e46.png)
![image](https://user-images.githubusercontent.com/57212002/177332960-b17c1a7d-5e2b-4e85-90b9-c2c3b79424ee.png)